Packs/FiltersAndTransformers/Scripts/ExtractEmailTransformer/ExtractEmailTransformer.yml

```diff
@@ -0,0 +1,15 @@
+comment: ExtractEmailTransformer
+commonfields:
+  id: ExtractEmailTransformer
+  version: -1
+name: ExtractEmailTransformer
+script: '-'
+tags:
+- Transformer
```

Comment:
missing definition of `value` as an argument
should it support a list value? if so, the code needs to be updated